packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -11,8 +11,16 @@ import { Boundary } from "../../common/boundary";
 import * as Classes from "../../common/classes";
 import { OVERFLOW_LIST_OBSERVE_PARENTS_CHANGED } from "../../common/errors";
 import { DISPLAYNAME_PREFIX, IProps } from "../../common/props";
+import { safeInvoke } from "../../common/utils";
 import { IResizeEntry, ResizeSensor } from "../resize-sensor/resizeSensor";
 
+/** @internal - do not expose this type */
```

Comment:
thoughts on doing this to reduce the exported surface area? it excludes it from the `.d.ts` files and required marking the state field similarly (line 84).

Reply:
🆒